src/test_sort.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import random
+
+
+EDGE_CASES = [[[100, 100, 100, 100, 100], [100, 100, 100, 100, 100]],
+              [[999999999999999, 0], [0, 999999999999999]],
+              [[1000000000, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 1000000000]],
+              [[], []],
+              [[1, 100, 1, 100, 1, 100, 1, 100, 1, 100],
+               [1, 1, 1, 1, 1, 100, 100, 100, 100, 100]]
+              ]
```

Comment:
Your edge cases should include sorting strings as well, or rather a list of strings.